python/looker_sdk/rtl/api_settings.py

```diff
@@ -93,23 +102,27 @@ def __init__(
         if sdk_version:
             self.agent_tag += f" {sdk_version}"
 
-    def read_config(self) -> Dict[str, str]:
+    def read_config(self) -> SettingsConfig:
         cfg_parser = cp.ConfigParser()
+        data: SettingsConfig = {
```

Comment:
If the user doesn't end up settings these they'll run into this [exception](https://github.com/looker-open-source/sdk-codegen/blob/d5b0c999727e6065723ac6becbb5c53e98550387/python/looker_sdk/rtl/auth_session.py#L136)